concent_api/core/message_handlers.py

```diff
@@ -428,21 +429,33 @@ def handle_send_force_payment(request, client_message: message.concents.ForcePay
     client_public_key       = decode_client_public_key(request)
     other_party_public_key  = decode_other_party_public_key(request)
     current_time            = get_current_utc_timestamp()
+    concent_rpc             = new_sci_rpc(settings.GETH_CONTAINER_ADDRESS, settings.CONCENT_ETHEREUM_ADDRESS, lambda tx: tx.sign(settings.CONCENT_ETHEREUM_PRIVATE_KEY))
+    # TEMPORARY: waiting until Concent will be synchronized
+    while not (concent_rpc.is_synchronized()):
```

Comment:
I know you have marked this as temporary but how it would work when completed ? Will there be some timeout ? If not, I think we should implement one on our side.